cardano-config/src/Cardano/Config/Partial.hs

```diff
@@ -212,14 +212,14 @@ checkComplete :: String -> Last a -> Either ConfigError a
 checkComplete name (Last x) = maybe (Left $ PartialConfigValue name) Right x
 
 --
--- The finalise* family of functions are supposed to be called at the very last stage
+-- The check* family of functions are supposed to be called at the very last stage
```

Comment:
@Jimbo4350, I'm sorry I wasn't clear -- I don't argue we are doing the checking part!  We clearly do.

But we _also_ do the `PartialCardanoConfiguration -> CardanoConfiguration` transformation (which I chose to call finalisation), which goes outside of the "checking" bill -- it's something beyond checking.

And this "something beyond" is important, as it marks transition from the internal representation to a usable value.
So that's why I chose "finalise" initially. 

So what I'm saying is that we probably want to pick a name that reflects that finalisation step, in some way.  If you don't like the particular word -- let's discuss other options.

Reply:
@deepfire How does `produceCardanoConfiguration` sound?

Reply:
Perhaps `mkCardanoConfiguration` is the better option, as it suggests it's the proper way to introduce values of this type.

Reply:
Decided on `mkCardanoConfiguration` on Slack.